tokio-console/src/main.rs

```diff
@@ -127,6 +127,13 @@ async fn main() -> color_eyre::Result<()> {
                     .0
                     .push(Span::styled(" PAUSED", view.styles.fg(Color::Red)));
             }
+            let dropped_events = state.async_ops_state().dropped_events();
+            if dropped_events > 0 {
+                header_text.0.push(Span::styled(
+                    format!(" dropped events: {}", dropped_events),
+                    view.styles.fg(Color::Red),
+                ));
+            }
```

Comment:
It looks like this will only display the number of dropped async ops. But, we also have separate counters of [dropped tasks](https://docs.rs/console-api/latest/console_api/tasks/struct.TaskUpdate.html#structfield.dropped_events) and [dropped resources](https://docs.rs/console-api/latest/console_api/resources/struct.ResourceUpdate.html#structfield.dropped_events). If we only display the number of dropped async ops, we won't inform the user if any new tasks or resources were dropped. We should probably also include those in the UI.

Reply:
Should it be a one number with the sum of three or a separate counter for each one of them?

Reply:
I would probably have a separate counter for each.